routes/discordactions.js

```diff
@@ -16,6 +18,7 @@ const router = express.Router();
 router.post("/groups", authenticate, checkIsVerifiedDiscord, validateGroupRoleBody, createGroupRole);
 router.get("/groups", authenticate, checkIsVerifiedDiscord, getAllGroupRoles);
 router.post("/roles", authenticate, checkIsVerifiedDiscord, validateMemberRoleBody, addGroupRoleToMember);
+router.post("/nickname", authenticate, checkIsVerifiedDiscord, changeNicknameOfUsers);
```

Comment:
Just asking, you sure its not a SUPERUSER-only route?

Reply:
I have added superuser route